src/thunderbots/software/ai/intent/intent.h

```diff
@@ -3,7 +3,12 @@
 #include <string>
 
 // TODO: should this be an enum instead?
+const static std::string CATCH_INTENT_NAME = "Catch Intent";
+const static std::string CHIP_INTENT_NAME = "Chip Intent";
+const static std::string DIRECT_VELOCITY_INTENT_NAME = "Direct Velocity Intent";
+const static std::string KICK_INTENT_NAME = "Kick Intent";
 const static std::string MOVE_INTENT_NAME = "Move Intent";
+const static std::string PIVOT_INTENT_NAME = "Pivot Intent";
```

Comment:
Keeping in line with what we do for primitives, these should be moved to the top of their respective intent `.cpp` files.